### ISSUE TITLE
www.seeks.fr cannot be reached, search is then no longer working.

### ISSUE THREAD

**Issue by Quix0r** [beniz/seeks#16]
I get the following message in my logfile:

```
Apr 27 21:51:00.974 7f1cb0ff9700 Error: curl error on url www.seeks.fr/find_bqc?output=json: Could not resolve host: www.seeks.fr
Apr 27 21:51:01.701 7f1cb3fff700 Error: Failed deserializing db_query_record from gzip_stream
Apr 27 21:51:01.701 7f1cb3fff700 Error: Failed deserializing db_query_record
Apr 27 21:51:01.701 7f1cb3fff700 Error: transmission or deserialization error fetching batch records for query seeks from seeks-project.info/search.php
Apr 27 21:51:01.747 7f1cb37fe700 Error: Failed deserializing db_query_record from gzip_stream
Apr 27 21:51:01.747 7f1cb37fe700 Error: Failed deserializing db_query_record
Apr 27 21:51:01.747 7f1cb37fe700 Error: transmission or deserialization error fetching batch records for query seeks from seeks-project.info/search_exp.php
```

**Comment by beniz** [comment]
Yes, it is down forever. There was a plan to host it elsewhere, need to look at it again. I can basically point the domain wherever someone is willing to host it.

**Comment by Quix0r** [comment]
How much traffic does it consume?

**Comment by beniz** [comment]
Not much, a query per second at max, the project is not popular enough anymore. A protection against googlebots and other common crawlers is best to avoid useless traffic.

**Comment by Quix0r** [comment]
Or can you OpenSource them? Means that everyone can host their own scripts? I can put them on my server, I have this PHP version there:

PHP 5.4.39-0+deb7u2

**Comment by beniz** [comment]
The historical php script is already in the repo:
https://github.com/beniz/seeks/tree/master/resources

**Comment by Quix0r** [comment]
So I basicy have to run a seeks proxy on my server as I understand it? Okay, I'm currently improving the script a lot as it seems really rudimentarily written ... :-)

**Comment by Quix0r** [comment]
Okay, it is now here:
https://seeks.mxchange.org/search.php (only self-signed, I know).

search_exp.php ist still needed?

**Comment by beniz** [comment]
I don't believe you need search_exp

**Comment by beniz** [comment]
your setup seems to work fine, though slowly, probably one of your search results source is slow.

**Comment by Quix0r** [comment]
Okay, I see it also being slow. I have activated some more sources.

**Comment by GreenLunar** [comment]
I use https://seeks.ru/ which seems to be one of the longest running Seeks nodes.

**Comment by Quix0r** [comment]
Hello @GreenLunar you can run your own seeks instance but you have to modify etc/seeks/cf-config (and create a backup file so you can easily restore it after you have updated seeks).

Then find the lines with "cf-peer" and replae all with:
cf-peer http://seeks.mxchange.org/search.php bsn
cf-peer http://seeks.ru/search.php bsn

Then you have my node and the other one and can benefit from both node's peer lists

**Comment by Quix0r** [comment]
I have changed cf-config and still I'm getting this:

```
Jun 22 16:01:11.019 7fd01dfc3700 Error: curl error on url http://www.seeks-project.info/search.php/search/txt/abc123/1963566387?lang=en: Connection time-out
```

**Comment by Quix0r** [comment]
Found another one:
_query-capture-config_

```
cross-post-url http://seeks.mxchange.org/search.php
```

**Comment by beniz** [comment]
seeks-project.info/search is down forever

**Comment by GreenLunar** [comment]
>  https://seeks.mxchange.org/search.php (only self-signed, I know).

@Quix0r, please make https://seeks.mxchange.org/ (Error 403) to redirect to https://seeks.mxchange.org/search.php

See also https://github.com/beniz/seeks/wiki/List-of-Web-Seeks-nodes#seeksmxchangeorg

**Comment by Quix0r** [comment]
It doesn't seem to work. I added a simple index.php with a redirect to search.php, not working. You end up in seeks proxy message.

Edit: Okay, the "header(Location: search.php); seems to mess around with seeks' internal handling of URLs. I have now setup a meta-refresh which seems to work. :)
